agency_client/Cargo.toml

```diff
@@ -16,7 +16,7 @@ lazy_static = "1.3"
 serde = "1.0.97"
 serde_json = "1.0.40"
 serde_derive = "1.0.97"
-reqwest = { version = "0.9.5" }
+reqwest = "0.9.5"
```

Comment:
Doesn't seem to revert reqwest version.

Reply:
It doesn't revert reqwest specifically, but it does revert dependencies which previous upgrade of reqwest has caused bringing in in lock file. When the reqwest was previously downgraded from 11 to 10, then from 10 to 9, in neither of these downgrades we reverted dependencies brought in by the first `9->11` upgrade.